qa/suites/rgw/sts/tasks/webidentity.yaml

```diff
@@ -0,0 +1,17 @@
+tasks:
+- tox: [ client.0 ]
+- keycloak:
```

Comment:
Somewhere here is where I would pass in the version (ex: `11.0.0`) so you're not hard coding it in keycloak.py